website/home/_posts/wheres_the_graph.md

```diff
@@ -0,0 +1,148 @@
+
+Refactoring becomes vastly simpler when dependencies are explicit and components are reusable. We've all been there—trying to modify one part of a complex workflow only to discover hidden dependencies that break something seemingly unrelated. With GenSX's tree approach, those dependencies are right there in the code.
+
+Our team's velocity dramatically increased when we switched to this model. Building reusable components feels natural, not forced. You're writing code the way you'd write any other TypeScript application, using patterns you already know.
+
+## Your Graph Is Still There (Just Better)
+
+Look, I'm not anti-graph. I'm anti-unnecessary complexity. GenSX still represents your workflow as a directed graph internally. We just give you a more intuitive interface for expressing it.
+
+In tree form, data dependencies between steps are crystal clear. Each child component only sees the output of its parent through explicit props or child functions. This explicitness means no mysterious global state, no tangled webs of dependencies, and no edge spaghetti that makes you question your career choices at 2 AM.
+
+## The Tree Advantage
+
+Trees aren't just simpler to read—they're also easier to test, debug, and reason about. Isolation is built-in because each component is a pure function of its inputs. Testing flows naturally since you can evaluate components independently. Debugging becomes straightforward because the call stack directly represents your workflow structure.
+
+And the kicker? Functional components can be published and shared on npm, creating an ecosystem of reusable workflow patterns. Try doing that with a framework dependent on global state and proprietary graph syntax.
+
+## But What About...?
+
+Every time I present this approach, someone inevitably asks: "But what about [complex pattern X]? Surely you need a graph for that!"
+
+In practice, we've found that the vast majority of workflows are simple linear chains—perfect for trees. A smaller percentage need basic branching, which is trivial with JS conditionals. And those complex patterns like cycles? They're handled elegantly through components and recursion.
+
+From simple RAG pipelines to complex agentic systems, we've yet to encounter a pattern that this model can't handle with grace. The rare edge cases where traditional graphs might shine are easily addressed with standard programming language features.
+
+## Embrace the Tree
+
+Next time you're designing a workflow system, resist the urge to reach for the graph API first. Ask yourself: could this be expressed as a tree? Could standard programming language features handle the more complex patterns?
+
+Your future self—trying to understand that workflow six months from now—will thank you. Your teammates who need to maintain your code will buy you coffee. And your production systems will be more robust for it.
+
+And maybe give GenSX a try. We've bet our workflow model on trees, and we haven't looked back.
```

Comment:
probably worth adding a link someone can click to sign up/get started

Reply:
Yeah, link to the github here would be good. There is a signup CTA at the bottom of the page